scrapy/templates/project/module/settings.py.tmpl

```diff
@@ -16,75 +16,75 @@ NEWSPIDER_MODULE = '$project_name.spiders'
 
 
 # Crawl responsibly by identifying yourself (and your website) on the user-agent
-#USER_AGENT = '$project_name (+http://www.yourdomain.com)'
+# USER_AGENT = '$project_name (+http://www.yourdomain.com)'
 
 # Obey robots.txt rules
 ROBOTSTXT_OBEY = True
 
 # Configure maximum concurrent requests performed by Scrapy (default: 16)
-#CONCURRENT_REQUESTS = 32
+# CONCURRENT_REQUESTS = 32
```

Comment:
hum, I'd rather leave as it is, because `#noqa` comments look really weird for those who aren't familiar with it. [At some point](https://github.com/PyCQA/pycodestyle/issues/476), not even Guido knew what `noqa` stands for 😆